### PR TITLE
Internal: Change Getting Started video and links to support Container release [ED-12069]

### PR DESCRIPTION
**Pull request by Barcov**

### CODE REVIEW

File: modules/shapes/widgets/text-path.php [hunk 1]
```diff
@@ -659,7 +659,7 @@ protected function render() {
 		// Add Text Path attributes.
 		$this->add_render_attribute( 'text_path', [
 			'class' => 'e-text-path',
-			'data-text' => esc_attr( $settings['text'] ),
+			'data-text' => htmlentities( esc_attr( $settings['text'] ) ),
```
Comment (hein-obox) [inline modules/shapes/widgets/text-path.php:662]: This seems to be unrelated. If so, please restore it to the original version.

Comment (Barcov) [reply modules/shapes/widgets/text-path.php]: Thanks

**Review by hein-obox — APPROVED** [review]
